static/usage/popover/customization/styling/demo.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Popover</title>
+  <link rel="stylesheet" href="../../../common.css" />
+  <script src="../../../common.js"></script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@6/dist/ionic/ionic.esm.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@6/css/ionic.bundle.css" />
+
+  <style>
+    ion-popover {
+      --background: rgba(40, 173, 218, 0.6);
+      --backdrop-opacity: 0.6;
```

Comment:
Any particular reason why you opted to use the backdrop opacity CSS variable instead of setting `opacity` on `::part(backdrop)`?

Reply:
Just to show that the variable exists, mostly 😆 I can change it if you like, although that makes me wonder why we have the variable in the first place 😄

Reply:
It's up to you. The variable was added before shadow parts were available in most browsers. Though with the shadow part it's redundant.

Reply:
So it turns out that `ion-popover` has a default value for `--backdrop-opacity` built-in: https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/popover/popover.ios.scss#L11 This overwrites setting `opacity` on the `ion-backdrop` directly, so setting a custom `--backdrop-opacity` on the popover is the cleanest way of doing things.